web/src/data/events.js

```diff
@@ -17,11 +17,21 @@ const EventTypes = Object.freeze({
 });
 
 class Event {
-  constructor(type, title, date, eventSchedule, externalLinks) {
+  constructor(
+    type,
+    title,
+    date,
+    eventSchedule = [],
+    location = "",
+    price = "",
+    externalLinks = undefined
+  ) {
```

Comment:
This shit is why we need typescript

Reply:
fair, but this is probably the only thing that will have to look like this. Nothing else is going to be rendered dynamically